PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskJetCoreEmcal.h

```diff
@@ -111,6 +113,11 @@ class AliAnalysisTaskJetCoreEmcal : public AliAnalysisTaskEmcalJet {
 	TH2F *fh2RPTC10; //!
 	TH2F *fh2RPTC20; //!
 
+	TH2F *fhTTPt; //!
```

Comment:
I'd suggest to use the new style comments that work for doxygen in both root 5 and 6. In that system, `//!` -> `//!<!` (and for streaming objects, `//` -> `///<`)

Reply:
Thanks for the suggestion Raymond, will do. For the moment, can you approve as it is (assuming testing succeeds), and I will fix as suggested later?

Reply:
Yes thanks for spotting! I also did the doxygen compatibility fixes where needed (now just on those which already have comments and transient members)

Reply:
Thanks!